lib/_chaestli.scss

```diff
@@ -202,3 +212,73 @@
     @include grid--define-col;
     @include grid--span($cfg);
 }
+
+/// Allows the grid debugging coloring the rows background showing the columns and the edges
+/// @param {number} $grid-cfg.gutter - The inner gutter size
+/// @param {number} $grid-cfg.edge - The edge clearance in any spacial unit
+/// @param {number} $cfg.columns - The amount of columns we want to debug
+/// @param {number} $cfg.column-color - The color used to debug the colors
```

Comment:
Typo: `columns` instead of `colors`

Reply:
```suggestion
/// @param {number} $cfg.column-color - The color used to debug the columns
```

Reply:
PRO TIP: @nirazul for typos you could simply use `suggestion` in order to speed up review

https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change